services/business/istio_config.go

```diff
@@ -38,12 +40,14 @@ func (in *IstioConfigService) GetIstioConfig(criteria IstioConfigCriteria) (mode
 		DestinationRules:    models.DestinationRules{},
 		ServiceEntries:      models.ServiceEntries{},
 		Rules:               models.IstioRules{},
+		QuotaSpecs:			 models.QuotaSpecs{},
+		QuotaSpecBindings:   models.QuotaSpecBindings{},
 	}
-	var gg, rr, dp, vs, dr, se []kubernetes.IstioObject
+	var gg, rr, dp, vs, dr, se, qs, qb []kubernetes.IstioObject
```

Comment:
You'll soon run out of available initials :)

Reply:
Nah, my next task will be to remove the old API, so I will release some of them :-)